lib/host/ui/unregister_ao_uis.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const debug = require('debug')('bfx:hf:host:register-ao-uis')
+const _isFunction = require('lodash/isFunction')
+
+const { PLATFORM = 'bitfinex' } = process.env
+
+/**
+ * Registers all order form layouts for the provided AO Host
+ *
+ * @param {Object} aoHost - algo order host
+ */
+module.exports = async (aoHost = {}, aoID) => {
```

Comment:
Hmm im not sure if this is the best way to handle this, maybe we should introduce an 'Active' flag similar to the AOs? The problem with this is that there isnt any state of the definition being de-registered. So if the AOs are re-registed to the UI via (https://github.com/bitfinexcom/bfx-hf-algo/blob/master/lib/host/ui/register_ao_uis.js) then previously de-registered AO will appear in the UI again

Reply:
This might cause a problem in the future when we start allowing the user to create new AOs and therefore promp a call to `register_ao_uis`

Reply:
Sorry, we need to delete this file, it's unused